pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -313,12 +317,18 @@ private boolean hasQuarkusExtensionMetadata(Path path) {
 
     private Set<String> gatherProblematicDeps() {
         Set<String> problemmaticDeps = new TreeSet<>();
+        String bomArtifactId = getBomArtifactId();
         problemmaticDeps.addAll(checkBomContents(".*/io/quarkus/quarkus-bom/.*\\.pom"));
-        problemmaticDeps.addAll(checkBomContents(".*/com/redhat/quarkus/quarkus-product-bom/.*\\.pom"));
+        if (bomArtifactId.equals("quarkus-product-bom")) {
```

Comment:
I think it would be a bit better to have the whole check in a separate method too, in case it gets changed. We're doing it in 3 places already now, it would be easy to miss one.